Frameworks/MQTTnet.AspnetCore/MQTTnet.AspnetCore.csproj

```diff
@@ -2,13 +2,14 @@
 
```

Comment:
I assume this is not required and can be reverted?

Reply:
True